concepts/05 Widgets/Scheduler/00 Getting Started with Scheduler/15 Edit Appointments.md

```diff
@@ -0,0 +1,92 @@
+Users can do the following actions on appointments:
+
+* Add new appointments
+
+* Change text and date fields in the details form
+
+* Change start and end time zones in the details form
+
+* Rese the appointment's rectangle to change the duration
+
+* Drag the appointment's rectangle to change start and end dates 
+
+* Remove appointments with the help of the appointment's tooltip.
+
+All of these operations - except time zone changing and appointment's dragging and resizing on iOS and Android - are enabled. To change the default settings, use the [editing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/) object.
+
+In this step, we will disable drag appointments' dragging and enable time zone editing. For this, we will set the **editing**.[allowDragging](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/#allowDragging) to `false` and editing.[allowTimeZoneEditing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/#allowTimeZoneEditing) to `true`.
+
+[note] The **Scheduler** displays appointment's time zone only when **editing**.**allowTimeZoneEditing** is `true`.
```

Comment:
Time zones have very little to do with editing. Look how much shorter this topic becomes if we remove all irrelevant information:

```suggestion
Edit operations are enabled by default. To disable an individual operation, assign **false** to its property in the [editing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/) object. In this tutorial, we use the [allowDragging](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/#allowDragging) property to disable appointment drag-and-drop.
```

Reply:
> Edit operations are enabled by default.

Not all edit operations are enabled be default. Why are time zone changing and appointment's dragging and resizing on iOS and Android irrelevant to the 'Edit Appointments' part of the tutorial?

Reply:
Because this is an entry-level tutorial. It doesn't have to encompass all aspects and caveats of every component feature. It must be simple.